core/src/main/java/org/testcontainers/dockerclient/UnixSocketClientProviderStrategy.java

```diff
@@ -9,13 +9,13 @@
 import java.nio.file.Paths;
 
 /**
- *
  * @deprecated this class is used by the SPI and should not be used directly
  */
 @Deprecated
 public final class UnixSocketClientProviderStrategy extends DockerClientProviderStrategy {
+
+    protected static final String SOCKET_LOCATION = "unix:///var/run/docker.sock";
     protected static final String DOCKER_SOCK_PATH = "/var/run/docker.sock";
```

Comment:
nit (reducing the changeset):
```suggestion
    protected static final String DOCKER_SOCK_PATH = "/var/run/docker.sock";
    private static final String SOCKET_LOCATION = "unix://" + DOCKER_SOCK_PATH;
```